server/src/main/java/com/testsigma/dto/TestStepDTO.java

```diff
@@ -78,7 +78,7 @@ public class TestStepDTO implements Cloneable, Serializable {
   private Long testDataId;
   private Integer testDataIndex;
   private String setName;
-
+  private TestCaseEntityDTO componentTestCaseEntity;
```

Comment:
@nixonakash01 What is the need for this field ?

Reply:
that is being used in WhileLoopStepProcessor         cstepEntity.setComponentTestCaseEntity(centity.getComponentTestCaseEntity());
and 
stepEntityFromGroup.setComponentTestCaseEntity(stepDTOFromGroup.getComponentTestCaseEntity());

Reply:
i verified the working and attached screenshots in the pr, just in case